source/dotnet/Measurements.WebApi/Controllers/MeasurementsController.cs

```diff
@@ -131,4 +131,21 @@ public async Task<IActionResult> GetAggregatedByMonthAsync([FromQuery] GetAggreg
             return NotFound(e.Message);
         }
     }
+
+    [MapToApiVersion(1.0)]
```

Comment:
Current version is v3
```suggestion
    [MapToApiVersion(3.0)]
```